consumptions/tests/test_views.py

```diff
@@ -1,32 +1,43 @@
-from django.test import TestCase, Client
+from django.test import TestCase
 from django.urls import reverse
-from consumptions.models import *
 import json
+from rest_framework import status
+from rest_framework.test import APIClient
+from consumptions.models import Product_Category, Product
+
 
 class TestViews(TestCase):
 
     def setUp(self):
-        self.client = Client()
-        self.prod_cat = Product_Category.objects.create(code='LL', name='landline')
-        self.prod = Product.objects.create(code='LL001', name='ADSL', category=self.prod_cat)
-        self.cons = Consumption.objects.create(product=self.prod, quantity=10)
+        self.client = APIClient()
 
     def test_index_get(self):
-        response = self.client.get(reverse('consumptions:index'))
-        self.assertEquals(response.status_code, 200)
-        self.assertTemplateUsed(response, 'index.html')
+        """Test retrieving the home page"""
+        res = self.client.get(reverse('consumptions:index'))
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
+        self.assertTemplateUsed(res, 'index.html')
 
     def test_consumption_table_get(self):
-        response = self.client.get(reverse('consumptions:consumption_list'))
-        self.assertEquals(response.status_code, 200)
-        self.assertTemplateUsed(response, 'consumption_list.html')
+        """Test retrieving a consumptions list"""
+        res = self.client.get(reverse('consumptions:consumption_list'))
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
+        self.assertTemplateUsed(res, 'consumption_list.html')
 
     def test_consumption_create_get(self):
```

Comment:
This is test name is very confusing. What did want to communicate with that `_get` suffix? This create view has nothing to do with an HTTP GET request or similar :thinking:

Reply:
i put the _get suffix to differentiate create  view from create action, but i can see it can be confusing

Reply:
i replaced _get with _view